### PR TITLE
[IOTDB-3482] Add IT for UDF management in new Cluster

### PR DESCRIPTION
**Pull request by Plutooooooo**

### CODE REVIEW

File: integration-test/src/test/java/org/apache/iotdb/db/it/udf/IoTDBUDFManagementIT.java [hunk 1]
```diff
@@ -16,77 +16,69 @@
...
-        CompressionType.UNCOMPRESSED,
-        null);
-    Class.forName(Config.JDBC_DRIVER_NAME);
+    enableSeqSpaceCompaction = ConfigFactory.getConfig().isEnableSeqSpaceCompaction();
+    enableUnseqSpaceCompaction = ConfigFactory.getConfig().isEnableUnseqSpaceCompaction();
+    enableCrossSpaceCompaction = ConfigFactory.getConfig().isEnableCrossSpaceCompaction();
+    ConfigFactory.getConfig().setEnableSeqSpaceCompaction(false);
+    ConfigFactory.getConfig().setEnableUnseqSpaceCompaction(false);
+    ConfigFactory.getConfig().setEnableCrossSpaceCompaction(false);
+    EnvFactory.getEnv().initBeforeClass();
```
Comment (JackieTien97) [inline integration-test/src/test/java/org/apache/iotdb/db/it/udf/IoTDBUDFManagementIT.java:68]:
```suggestion
    EnvFactory.getEnv().initBeforeTest();
```
If you mean that you need to do this setUp() method every time test case is running, you should call `initBeforeTest`.

File: integration-test/src/test/java/org/apache/iotdb/db/it/udf/IoTDBUDFManagementIT.java [hunk 2]
```diff
@@ -16,77 +16,69 @@
...
   }
 
   @After
-  public void tearDown() throws Exception {
-    EnvironmentUtils.cleanEnv();
+  public void tearDown() {
+    EnvFactory.getEnv().cleanAfterClass();
```
Comment (JackieTien97) [inline integration-test/src/test/java/org/apache/iotdb/db/it/udf/IoTDBUDFManagementIT.java:73]:
```suggestion
    EnvFactory.getEnv().cleanAfterTest();
```

File: integration-test/src/test/java/org/apache/iotdb/db/it/udf/IoTDBUDFManagementIT.java [hunk 3]
```diff
@@ -279,86 +253,58 @@ public void testDropFunction2() { // drop
...
-        assertTrue(throwable.getMessage().contains("Built-in function"));
+        assertTrue(
+            throwable.getMessage().contains("Built-in function ABS can not be deregistered"));
       }
-      statement.execute("INSERT INTO root.vehicle.d1(time, s1) VALUES(1, -10.0)");
-      ResultSet rs = statement.executeQuery("SELECT ABS(s1) FROM root.vehicle.d1");
-      Assert.assertTrue(rs.next());
-      Assert.assertEquals(1, rs.getLong(1));
-      Assert.assertEquals(10.0F, rs.getFloat(2), 0.00001);
-      Assert.assertFalse(rs.next());
```
Comment (JackieTien97) [inline integration-test/src/test/java/org/apache/iotdb/db/it/udf/IoTDBUDFManagementIT.java:297]: Why delete these assert

File: integration-test/src/test/java/org/apache/iotdb/db/it/udf/IoTDBUDFManagementIT.java [hunk 4]
```diff
@@ -16,77 +16,69 @@
...
 
   @Test
   public void testCreateReflectShowDrop() {
-    try (Connection connection =
-            DriverManager.getConnection(
-                Config.IOTDB_URL_PREFIX + "127.0.0.1:6667/", "root", "root");
+    try (Connection connection = EnvFactory.getEnv().getConnection();
         Statement statement = connection.createStatement()) {
       statement.execute("create function udf as 'org.apache.iotdb.db.query.udf.example.Adder'");
       statement.execute("select udf(*, *) from root.vehicle");
```
Comment (JackieTien97) [inline integration-test/src/test/java/org/apache/iotdb/db/it/udf/IoTDBUDFManagementIT.java:84]: Why executing query sql here, not verifying the correctness?

**Review by JackieTien97 — CHANGES_REQUESTED** [review]

File: integration-test/src/test/java/org/apache/iotdb/db/it/udf/IoTDBUDFManagementIT.java (hunk 3 above)
Comment (Plutooooooo) [reply integration-test/src/test/java/org/apache/iotdb/db/it/udf/IoTDBUDFManagementIT.java]: CI was not stable when calling insert yesterday, these are added back now.

File: integration-test/src/test/java/org/apache/iotdb/db/it/udf/IoTDBUDFManagementIT.java (hunk 4 above)
Comment (Plutooooooo) [reply integration-test/src/test/java/org/apache/iotdb/db/it/udf/IoTDBUDFManagementIT.java]: I think query here is to make sure that udf function itself is created correctly, So in this IT we don't need to check the result of this sql.

File: integration-test/src/test/java/org/apache/iotdb/db/it/udf/IoTDBUDFManagementIT.java (hunk 2 above)
Comment (Plutooooooo) [reply integration-test/src/test/java/org/apache/iotdb/db/it/udf/IoTDBUDFManagementIT.java]: Fixed.

File: integration-test/src/test/java/org/apache/iotdb/db/it/udf/IoTDBUDFManagementIT.java (hunk 1 above)
Comment (Plutooooooo) [reply integration-test/src/test/java/org/apache/iotdb/db/it/udf/IoTDBUDFManagementIT.java]: Fixed.

**Pull request merged**